mautrix_signal/__main__.py

```diff
@@ -196,5 +202,39 @@ async def manhole_global_namespace(self, user_id: UserID) -> Dict[str, Any]:
             "Puppet": Puppet,
         }
 
+    async def _notify_bridge_blocked(self, is_blocked: bool = True) -> None:
+        msg = self.config["bridge.limits.block_ends_notification"]
+        if is_blocked:
+            msg = self.config["bridge.limits.block_begins_notification"]
+            next_notification = self.last_blocking_notification + self.config["bridge.limits.block_notification_interval_seconds"]
+            # We're only checking if the block is active, since the block-end notification will not be resent and we want it ASAP
```

Comment:
This line made me wonder if we should use the term unblock instead of block-end, which I didn't understand at first.
```suggestion
            # We're only checking if the block is active, since the unblock notification will not be resent and we want it ASAP
```